packages/components/src/components/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,305 @@
+
+  describe('Success variant', () => {
+    describe('Light mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'light')
+
+        component = render(
+          <Alert variant="success" header={headerText} testID={testID} />,
+        )
+
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#ecf3ec')
+        expect(borderLeftColor).toBe('#008817')
+        expect(getTextColor(component)).toBe('#3d4551')
+        expect(getIconName(component)).toBe('Check')
+      })
+    })
+
+    describe('Dark mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'dark')
+        component = render(
+          <Alert variant="success" header={headerText} testID={testID} />,
+        )
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#19311e')
+        expect(borderLeftColor).toBe('#5e9f69')
+        expect(getIconName(component)).toBe('Check')
+        expect(getTextColor(component)).toBe('#f0f0f0')
+      })
+    })
+  })
+
+  describe('Warning variant', () => {
+    describe('Light mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'light')
+
+        component = render(
+          <Alert variant="warning" header={headerText} testID={testID} />,
+        )
+
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#faf3d1')
+        expect(borderLeftColor).toBe('#ffbe2e')
+        expect(getTextColor(component)).toBe('#3d4551')
+        expect(getIconName(component)).toBe('ExclamationTriangle')
+      })
+    })
+
+    describe('Dark mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'dark')
+        component = render(
+          <Alert variant="warning" header={headerText} testID={testID} />,
+        )
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#5c4809')
+        expect(borderLeftColor).toBe('#face00')
+        expect(getIconName(component)).toBe('ExclamationTriangle')
+        expect(getTextColor(component)).toBe('#f0f0f0')
+      })
+    })
+  })
+
+  describe('Error variant', () => {
+    describe('Light mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'light')
+
+        component = render(
+          <Alert variant="error" header={headerText} testID={testID} />,
+        )
+
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#f8dfe2')
+        expect(borderLeftColor).toBe('#b50909')
+        expect(getTextColor(component)).toBe('#3d4551')
```

Comment:
Very nitpicky, but it's a little weird that for all variants the light/dark mode `expect`s swap the order of TextColor/IconName.

Reply:
Fixed